CHANGELOG.md

```diff
@@ -23,7 +25,6 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 - Use `c.FullPath()` method to set `http.route` attribute in `go.opentelemetry.io/contrib/instrumentation/github.com/gin-gonic/gin/otelgin`. (#5734)
 - The double setup in `go.opentelemetry.io/contrib/instrumentation/net/http/httptrace/otelhttptrace/example` that caused duplicate traces. (#5564)
 - Out-of-bounds panic in case of invalid span ID in `go.opentelemetry.io/contrib/propagators/b3`. (#5754)
-- Do not panic if a zero-value `SpanProcessor` is used from `go.opentelemetry.io/contrib/processors/baggage/baggagetrace`. (#5811)
```

Comment:
Why are this and other entries below removed?

Reply:
These changes are removed from the `baggagetrace` package. The package is reverted to the last released version. The idea being instead of having users upgrade to the new API of a deprecated package, have them upgrade to the replacement package instead.

Reply:
Should they be changed to refer to baggagecopy?  Just seems odd that they wouldn't be in the release notes at all

Reply:
They weren't change in `baggagecopy` though. That is a new module.

Reply:
@dashpole would it help if I list the differences between the two modules in the "Add" changelog entry?

Reply:
That would be fine, or you could list both modules in the changelog.  Isn't a huge deal because the module is relatively new.  But for things like "fix panic", it is nice to know what release and what module it is fixed in

Reply:
I've updated the changelog to include the new function migration.

I left out the panic fix though. Looking through the commit history, that panic has not actually been released. [It was merged](https://github.com/open-telemetry/opentelemetry-go-contrib/commit/85969a37b9d3f52f7858dda24eaaf769379c9923) on June 6th.

Reply:
We might need a way to lock markdown content for released sections in CI. So, the PR after the release won't accidentally change the released content.